lib/controller/page/profile_page_controller.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/view/dialog/confirm_logout_dialog.dart';
+import 'package:idea_tracker/view/page/splash_page.dart';
+
+class ProfilePageController extends ChangeNotifier {
```

Comment:
Add a callback function the page view can listen to

`Function onLogoutSuccess;`

Reply:
When the function is called, the page view should navigate to the landing page